### PR TITLE
Introduce flows specific to objects.

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch segregates flows specific to an object
under the "flows" of that object instead of the
global namespace.

tendrl-spec: segragate_object_specific_flows
tendrl-bug-id: Tendrl/ceph_integration#66
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@shtripat @nthomas-redhat @anivargi @brainfunked Please review

**Review by shtripat — APPROVED** [review]
LGTM

File: tendrl/ceph_integration/manager/tendrl_definitions_ceph.py [hunk 1]
```diff
@@ -61,6 +46,22 @@
...
+            - tendrl.ceph_integration.objects.pool.atoms.delete
+          description: "Delete Ceph Pool"
+          enabled: true
+          inputs:
+            mandatory:
+              - Pool.pool_id
+              - Tendrl_context.sds_name
+              - Tendrl_context.sds_version
+              - Tendrl_context.cluster_id
+          run: tendrl.ceph_integration.flows.delete_pool.DeletePool
```
Comment (nnDarshan) [inline tendrl/ceph_integration/manager/tendrl_definitions_ceph.py]: tendrl.ceph_integration.objects.Pool.flows.delete_pool.DeletePool

Comment (shtripat) [reply tendrl/ceph_integration/manager/tendrl_definitions_ceph.py]: As we talked and discussed, as part of this PR the parsing logic for the flows would require changes as we should be able to figure out if a flow is generic one or specific to an object. Lets take that change as part of this PR.

**Review by r0h4n — APPROVED** [review]

**Pull request merged**